pkg/apis/maistra/v2/grafana.go

```diff
@@ -27,7 +27,7 @@ type GrafanaInstallConfig struct {
 	Service *ComponentServiceConfig `json:"service,omitempty"`
 	// Persistence configures a PersistentVolume associated with the grafana installation
 	// .Values.grafana.persist, true if not null
-	// XXX: capacity is not supported in the charts, hard coded to 5Gi
+	// XXX: Capacity is configurable, default: 5Gi
```

Comment:
You can remove this line. These XXX lines were added as TODOs by Rob when he initially implemented the v2 SMCP.